content/r/concepts/data-frames/terms/ncol/ncol.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'ncol()'
+Description: 'Returns the number of columns in a data frame.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-r'
+  - 'paths/computer-science'
+---
+
+The **`ncol()`** function returns the number of columns in a data frame.
+
+## Syntax
+
+```pseudo
+ncol(df)
+```
+
+- `df`: The data frame to call the function on.
+
+## Example
+
+Suppose there exists a `patients.csv` file with the following information:
```

Comment:
```suggestion
The following example uses a  [CSV file](https://www.codecademy.com/resources/docs/r/csv-files) named `"patients.csv"` file with the following information:
```

I noticed this from the `colnames.md` file, but should we surround in backticks because it's a file name yet keep the quotes because it's passed into `read.csv()` as a string?